docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -195,6 +195,25 @@ def downstream_asset():
 
 In this example, Dagster doesn’t need to load data from `upstream_asset` to successfully compute the `downstream_asset`. While `downstream_asset` does depend on `upstream_asset`, the key difference with `non_argument_deps` is that data isn’t being passed between the functions. Specifically, the data from the `sugary_cereals` table isn't being passed as an argument to `downstream_asset`.
 
+#### Conditional materialization
```

Comment:
Nitpick: I think this should go at the bottom of this section (i.e. after "graph-backed assets" etc., because I think it's likely to be used less often than the other functionality)